src/popup.js

```diff
@@ -270,3 +270,12 @@ function create_csv(parameters_array){
     }
     return csv_file;
 }
+function create_json(parameter_array){
```

Comment:
minor thing: add an empty line between the two functions, and also add a comment similar to the export_csv

Reply:
Hey @inuyasha82 
Did a new commit https://github.com/inuyasha82/chrome_parameters/pull/67/commits/41a25511c265c90d5015528896415a87803bd0d2 addressing the comments.